### PR TITLE
Mesh Picnic Event

### PR DESCRIPTION
**Pull request by dasanchez**
This is the event post for the Mesh Picnic on September 29, 2018.
See Issue #230 for reference.

### CODE REVIEW

**Review by darkdrgn2k — APPROVED** [review]

File: _posts/2018-09-29-mesh-picnic.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+slug: mesh-picnic
+text: 'Mesh Outdoors with Toronto Mesh and Free Geek'
+location: Free Geek Toronto, 180 Sudbury St
+locationLink: https://osm.org/go/ZX6BI0AGy?node=4042949045
+date: 2018-09-29
+startTime: '13:00'
+endTime: '16:00'
+---
+
+We are having our first Mesh Picnic! [FreeGeek Toronto](https://www.freegeektoronto.org/) is hosting.
```
Comment (garrying) [inline _posts/2018-09-29-mesh-picnic.md]: `FreeGeek`->`Free Geek`

File: _posts/2018-09-29-mesh-picnic.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+**Technology showcase**  
+- Toronto Mesh nodes in action
+- File sharing through the [InterPlanetary File System](https://ipfs.io/) (IPFS)
+- Proof-of-concept long-range wireless link
+
+Bring a laptop or smartphone to join the mesh network!
+
+**Indoor activities**  
+- Computer components breakdown
+- FreeGeek Sale
```
Comment (garrying) [inline _posts/2018-09-29-mesh-picnic.md]: `FreeGeek` ->`Free Geek`

**Review by makew0rld — APPROVED** [review]
Looks good! Once we have some posters maybe we could add a picture of one to this post.

**Comment by ryan-fgt** [comment]
Here's the write up from the Free Geek event posting (please comment to let me know if any wording needs to be changed): 

Free Geek Toronto is hosting Toronto Mesh (tomesh.net) for what we're calling a picnic! So much of what we do is indoor, but on Sept 29th we'll be exploring the great outdoors (or the parks right beside Free Geek). Come learn about mesh networks, computer hardware, and some of the Open Source Software that makes it all work.

What should you expect?

Technology Showcase
Toronto Mesh nodes in action
File sharing through the InterPlanetary File System (IPFS)
Proof-of-concept long-range wireless link

Bring a laptop or smartphone to join the mesh network!

Indoor activities
Computer Disassembly and Hardware Check
Demonstrations of Free and Open Source Software
A 1 Day Sale on Desktop and Thrift Shop items!

Light refreshments will be available.

**Comment by makew0rld** [comment]
@ryan-fgt 

>Bring a laptop or smartphone to join the mesh network!

It might be more accurate to say "to take part in some demos!", since they won't be able to actually join the mesh, only access some services as a client. It's not an big deal though, just me being technical.

**Comment by ryan-fgt** [comment]
@makeworld-the-better-one no that makes sense. How about: "Bring a laptop or smartphone to take part in some demos of a mesh network!"

**Comment by darkdrgn2k** [comment]
would some one using the free wifi at the end of a mesh network consider participating in a mesh?

**Comment by dasanchez** [comment]
@garrying @ryan-fgt @makeworld-the-better-one @darkdrgn2k thanks for the input! I updated the post.

**Pull request merged**